index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <meta charset='UTF-8'/>
+    <meta name="description"/>
+    <title>Sedona cosik</title>
+  </head>
+
+  <body>
+    <div class="wrapper"><!-- main page -->
+      <header class="page-header">
+        <div calss="page-header__backgroun-img">
+          <nav class="main-nav"><!-- nav menu-->
+            <ul class="main-nav__list">
+              <li class="main-nav__item main-nav__item--active"><a href="index.html">Home</a></li>
+              <li class="main-nav__item"><a href="photos.html">Photos and videos</a></li>
+              <li class="main-nav__item"><a href="form.html">Feedback Form</a></li>
+              <li class="main-nav__item">HTML Academy</li>
+            </ul>
+            <button href="" class="main-nav__toggle main-nav__toggle--close">
+              <img src="" alt="Menu button" class="main-nav__button">
+            </button>
+            <img src="" class="main-nav__logo" alt="Sedona logo"> <!-- LOGO -->
+          </nav><!-- nav menu ends-->
+
+          <ul class="intro-img">
+            <li><img src="" class="intro-img--top" src="/" alt="Welcome text"></li>
+            <li><img src="" class="intro-img--middle" src="/" alt="Sedona"></li>
+            <li><img src="" class="intro-img--bottom" src="/" alt="Because the Grand Canyon sucks."></li>
+          </ul>
+        </div>
+      </header>
+
+      <section class="intro-text">
+        <div class="inner">
+          <h2 class="intro__header">SEDONA - a small town in the ARIZONA, deserves more!</h2>
+          <p class="intro-paragraph__text">Consider 5 reasons why Sedona is cooler than Grand Canyon!</p>
+        </div>
+      </section>
+
+      <section class="reasons">
+        <div class="reasons__inner reasons__inner--blue">
+          <h2 class="reasons__header">The Present Town</h2>
+          <p class="reasons-paragraph__text">— №1 —</p>
+          <p class="reasons-paragraph__text">Sedona is not a tourist attraction, the life floows in here in it own way.</p>
+          <div class="reasons-one__background-img"></div>
```

Comment:
We have several `reasons` block how are you going to set different background images to them? Also while we are using image replacement it's nice to have a text inside telling what's on image.